app/cmd/server.go

```diff
@@ -78,6 +88,26 @@ func start(vol volume.Volume) error {
 		shutdownCh <- err
 	}()
 
+	go func() {
```

Comment:
A commont for another part.

```go
	go func() {
		sig := <-sigs
		logger.Infof("share manager received signal %v to exit", sig)
		manager.Shutdown()
	}()

	return <-shutdownCh
```

Why we did not `close(shutdownCh)` after `manager.Shutdown()`.

Reply:
Addressed. Thank you.